### PR TITLE
feat: Log server url

### PR DESCRIPTION
**Pull request by BrunnerLivio**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?
Currently, the logs do not display where the server is running on

## What is the new behavior?
Logs the NestJS server url using `NestApplication.getUrl()`.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/lib/application/files/js/src/main.js [hunk 1]
```diff
@@ -4,5 +4,9 @@ import { AppModule } from './app.module';
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   await app.listen(3000);
+
+  const url = await app.getUrl();
+  // tslint:disable-next-line: no-console
```
Comment (BrunnerLivio) [inline src/lib/application/files/js/src/main.js]: We can bypass this error message by using
`process.stdout.write(...)` - but I thought `console.log()` more well-known

**Comment by kamilmysliwiec** [comment]
I like the idea of adding this log message to the existing samples (in the `nestjs` repo), but I don't think we should do it for the `nest new` schematic, especially when it requires an additional "ignore tslint" line. The `bootstrap()` function should be as minimal as possible and stay cohesive between different resources (starter project, `new` schematic, documentation, `nest g app` schematic).

**Comment by BrunnerLivio** [comment]
@kamilmysliwiec 
> adding this log message to the existing samples

The point is that newcomers shall know where the Nest app runs without reading the documentation. Most newcomers don’t look into the samples. I often switch between different frameworks (Angular, React, GastbyJS, etc.) and it happens that I forget which port NestJS runs on. It is a bit unpleasant to open up `main.ts` to check it.

> especially when it requires an additional "ignore tslint" line
Agree.

> stay cohesive between different resources
I would update it on all relevant resources ofc, but I want to get a approval on this PR before continuing the effort.

Other option may be including the log message into the CLI (as Angular CLI does it), though not quite sure how to approach it, since it shall be synched with `NestApplication.getUrl()`

**Comment by kamilmysliwiec** [comment]
> Most newcomers don’t look into the samples. I often switch between different frameworks (Angular, React, GastbyJS, etc.) and it happens that I forget which port NestJS runs on. It is a bit unpleasant to open up main.ts to check it.

As long as I understand the necessity for such a feature in the front-end space, where you need am additional web server which will serve your application so you can preview it and track the development progress (+ this server is usually shipped as an external library [webpack server] or built-in into the CLI [Angular] where you can only control the port by setting a flag [but first you have to find this information in the docs] which makes it fairly complex), server-side apps should be treated differently. In our case, you have full control over where and how (on which port) you will run your application. `main.ts` is an entry point for your application (like `index.html` for any website) and contains this information (specified port) already. Adding an extra console log doesn't make sense and would be useful only for those people who have never even tried creating a backend application (which means they will very likely be following the documentation which explicitly tells what to do and how to access the application). If we added something like this to the starter project, a lot of people (including me) would immediately remove these lines after initializing the application to get rid of redundant information.

**Comment by kamilmysliwiec** [comment]
I've spent a lot of time thinking about this and finally decided not to perform any changes to the `main.ts` (and hence, remain consistent with the current convention). I'm thankful for your contribution and I'm sorry that it won't be merged :(

**Pull request closed without merging**